### PR TITLE
Add a nightly bundle test Dockerfile

### PR DESCRIPTION
**Pull request by danielBelenky**
This image will be used to run tests against the nightly bundle
in openshift-ci.

Signed-off-by: Daniel Belenky <dbelenky@redhat.com>

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: deploy/nightly-bundle/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
```
Comment (danielBelenky) [inline deploy/nightly-bundle/deploy.sh:2]: Just a note here: this is a hard-link to deploy_marketplace.sh

File: deploy/nightly-bundle/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+PACKAGE="${PACKAGE:-kubevirt-hyperconverged}"
+CS_SOURCE="${CS_SOURCE:-hco-catalogsource}"
+
+TARGET_NAMESPACE="${TARGET_NAMESPACE:-kubevirt-hyperconverged}"
+MARKETPLACE_NAMESPACE="${MARKETPLACE_NAMESPACE:-openshift-marketplace}"
+HCO_REGISTRY_IMAGE="${HCO_REGISTRY_IMAGE:-quay.io/kubevirt/hco-container-registry:latest}"
+POD_TIMEOUT="${POD_TIMEOUT:-360s}"
+HCO_VERSION="${HCO_VERSION:-1.0.0}"
+HCO_CHANNEL="${HCO_CHANNEL:-1.0.0}"
```
Comment (nunnatsa) [inline deploy/nightly-bundle/deploy.sh:19]: Please use `1.1.0`

Comment (danielBelenky) [reply deploy/nightly-bundle/deploy.sh:19]: @nunnatsa  this is a hard link to `deploy_marketplace.sh`, not a new file so if possible, I'd rather not changing it right now...

**Comment by danielBelenky** [comment]
/hold

I see that openshift-ci allows me to rehearse on this PR... let's wait before merging it.

**Comment by tiraboschi** [comment]
just two new files for a parallel effort, no real code changes.
We can ignore CI.

**Pull request merged**